diff --git a/inference_utils.py b/inference_utils.py
index b5be6f1..502a758 100755
--- a/inference_utils.py
+++ b/inference_utils.py
@@ -105,7 +105,8 @@ def save_video(video, save_dir, file_name, fps=16.0, drop_prefix_cnt=0):
         tpth = os.path.join(temp_dir, "%06d.png" % (fid + 1))
         cv2.imwrite(tpth, frame[:, :, ::-1])
     tmp_path = os.path.join(save_dir, "tmp.mp4")
-    cmd = f"/usr/bin/ffmpeg -y -f image2 -framerate {fps} -i {temp_dir}/%06d.png \
+    ffmpeg = os.environ.get("FFMPEG_PATH", "ffmpeg")
+    cmd = f"{ffmpeg} -y -f image2 -framerate {fps} -i {temp_dir}/%06d.png \
      -vcodec libx264 -crf 17 -pix_fmt yuv420p {tmp_path}"
     status, output = subprocess.getstatusoutput(cmd)
     if status != 0:
